### PR TITLE
Refactor emulator project.

### PR DESCRIPTION
**Pull request by Zangetsu38**
- rename emulator project and folder to vita3k.
- Update few submodule :
glslang -> tag 7.11.3214
elfio -> last master
imgui -> tag v1.72
microprofile -> last master tag v3.1
pugixml -> last master
stp -> last master
unicorn-src -> last master
vita-headers -> last master
yaml-cpp -> last master

# Result
- Before
image

- After
image

### CODE REVIEW

**Comment by VelocityRa** [comment]
This should be rebased/reviewed _after_ #551 is merged.

File: vita3k/gui/src/gui.cpp [hunk 1]
```diff
@@ -82,9 +82,9 @@ static void init_style() {
     style->Colors[ImGuiCol_Header] = ImVec4(1.00f, 1.00f, 0.00f, 0.50f);
     style->Colors[ImGuiCol_HeaderHovered] = ImVec4(1.00f, 1.00f, 0.00f, 0.30f);
     style->Colors[ImGuiCol_HeaderActive] = ImVec4(1.00f, 1.00f, 0.00f, 0.70f);
-    style->Colors[ImGuiCol_Column] = ImVec4(0.56f, 0.56f, 0.58f, 1.00f);
-    style->Colors[ImGuiCol_ColumnHovered] = ImVec4(0.24f, 0.23f, 0.29f, 1.00f);
-    style->Colors[ImGuiCol_ColumnActive] = ImVec4(0.56f, 0.56f, 0.58f, 1.00f);
+    style->Colors[ImGuiCol_Separator] = ImVec4(0.56f, 0.56f, 0.58f, 1.00f);
```
Comment (VelocityRa) [inline vita3k/gui/src/gui.cpp]: I don't think this is right, are columns called seperators now? doesn't make much sense

Comment (Zangetsu38) [reply vita3k/gui/src/gui.cpp]: is explain by admin of imgui, have just renomed function

**Comment by VelocityRa** [comment]
Thanks, merging this.
Let's hope things don't break.

**Pull request merged**